hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/FSQueueConverter.java

```diff
@@ -215,6 +219,19 @@ private void emitPreemptionDisabled(String queueName, FSQueue queue) {
     }
   }
 
+  public void emitDefaultUserLimitFactor(Collection<FSLeafQueue> fsLeafQueue, CapacitySchedulerConfiguration config) {
+    fsLeafQueue
```

Comment:
There is no need to iterate through the queues, as the convertQueueHierarchy does that. Calling this method there and checking if the queue that we're currently converting is a leafQueue should achieve what we're trying to do.